mcstatus/protocol/connection.py

```diff
@@ -683,5 +701,13 @@ async def __aenter__(self) -> Self:
         await self.connect()
         return self
 
-    async def __aexit__(self, *_) -> None:
+    async def __aexit__(
+        self,
+        exc_type: type[BaseException] | None,
+        exc_val: BaseException | None,
+        exc_tb: TracebackType | None,
+    ) -> Literal[False]:
         self.close()
+        # Return false, we don't want to suppress
+        # exceptions raised in the context
+        return False
```

Comment:
Why make this change here? This PR doesn't seem to be the right place for it, but also, it's not very necessary to write out all of these variables when they aren't being used. Might even cause some linters/language servers to report warnings due to unused variables.

Reply:
Basically because mypy was mad that the function was untyped

Reply:
(In case, if you are waiting for a review, friendly reminder that we still need to resolve this conversation)

Reply:
Reverted changes as of 9676b71